adafruit_ads1x15/ads1x15.py

```diff
@@ -131,6 +141,25 @@ def gains(self) -> List[float]:
         g.sort()
         return g
 
+    @property
+    def compqueue(self) -> int:
+        """The ADC Comparator Queue."""
+        return self._compqueue
+
+    @compqueue.setter
+    def compqueue(self, compqueue: int) -> None:
+        possible_compqueues = self.compqueues
+        if compqueue not in possible_compqueues:
+            raise ValueError("Comparator Queue must be one of: {}".format(possible_compqueues))
+        self._compqueue = compqueue
+
+    @property
+    def compqueues(self) -> List[int]:
+        """Possible gain settings."""
+        g = list(_ADS1X15_CONFIG_COMP_QUEUE.keys())
+        g.sort()
+        return g
```

Comment:
I don't think you need to provide this. Just have them in the error message.

Reply:
The data_rate and gain settings were handled like this with respective *s property getters, so I did it this way too. I can change it, but I feel like settings with lists of possibilities should be handled the same way.